avd_docs/aws/cloudtrail/AVD-AWS-0015/CloudFormation.md

```diff
@@ -3,7 +3,7 @@ Enable encryption at rest
 
 ```yaml---
 Resources:
-  BadExample:
+  GoodExample:
```

Comment:
ditto

Reply:
Similar to my previous comment, the policy script has been updated as part of this PR and this should say GoodExample:
https://github.com/aquasecurity/defsec/blob/master/rules/cloud/policies/aws/cloudtrail/enable_at_rest_encryption.cf.go